assets/RNAseq_report.Rmd

```diff
@@ -456,8 +462,14 @@ if (params$input_type == "featurecounts") {
         #-->the sample name is in the subfolder name, not in the file name!
         files <- file.path(gsub("/$", "", params$path_gene_counts), qbicCodes, "quant.sf")
         if (!(all(file.exists(files)))) {
-            stop("DESeq2.R could not find all of the specified quant.sf files!")
-        }
+            missing_files <- "DESeq2.R could not find all of the specified .genes.results files! Please check that for all QBiC codes in the metadata sheet, a quant.sf file is present in the gene_counts folder and make sure that no typos exist. The following file(s) could not be found: "
```

Comment:
Here it should say in both places `quant.sf` files (not .genes.results). And could you also specify: `<QBiC-code>.genes.results`?